spec/requests/api/v1/usermedia/usermedia_request_spec.rb

```diff
@@ -30,4 +30,81 @@
       expect(user.user_medias.last.list.name).to eq "Want to Watch"
     end
   end
+
+  describe 'delete user_media/id' do
+    it 'deletes user_media for a user' do
+      user = create(:user) 
+      list_name = 'currently watching'
+      list = user.lists.find_by("name ILIKE ?", list_name)
+      user_media = create(:user_media, media_id: 3173903) 
```

Comment:
I don't remember my ILIKE rules, is it just case insensitivity or will it match partial results? I'm just concerned about how similar some of our list names are.